src/libPMacc/PMaccConfig.cmake

```diff
@@ -29,7 +29,7 @@
 ###############################################################################
 # PMacc
 ###############################################################################
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
we just updated the requirements to `3.3` yesterday, since a necessary [C++11 fix in FindCUDA.cmake](https://github.com/Kitware/CMake/blob/v3.3.0/Modules/FindCUDA.cmake#L1271-L1282) was first added in that version.

Therefore, now keep `3.3` when you rebase.